.github/workflows/code-coverage.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Get composer cache directory
         id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

Comment:
Interesting to note that there are a few variations of how dependencies are cached

- sometimes the name of the output is `dir`, sometimes `directory`
- sometimes the value is `$(composer config cache-files-dir)`, sometimes `$(composer config cache-dir)`

See #545.